src/core/backend.cc

```diff
@@ -40,6 +40,20 @@
 
 namespace nvidia { namespace inferenceserver {
 
+Status
+InferenceBackend::ValidateOutputName(const std::string& name) const
```

Comment:
After going through `NormalizeRequestHeader()`, I found an existing [`GetOutput()`](https://github.com/NVIDIA/tensorrt-inference-server/blob/master/src/core/backend.cc#L58-L71) function which may come in handy.